### ISSUE TITLE
Add php 7.1 and 7.2 support

### ISSUE THREAD

**Issue by Sentoki** [sidkshatriya/dontbug#12]
Is any plans do that?

**Comment by sidkshatriya** [comment]
I don't have plans at the moment. Can you try with PHP 7.0 -- let me know if you have any problems there. If there are more requests for PHP 7.1/7.2 I might consider...

**Comment by Sentoki** [comment]
Yep, some problem here: https://github.com/sidkshatriya/dontbug/issues/13

**Comment by massimiliano-brocchini** [comment]
+1 requesting php 7.2 support

**Comment by Mte90** [comment]
without support for the latest php version is not useful that amazing project :-(

**Comment by bgnih** [comment]
+1 PHP 7.2 support

**Comment by adriancofie** [comment]
This would be great with 7.2 support.

**Comment by honsa** [comment]
Would be great, it looks like this nice project is dead if you not support actual php versions. 

The idea is very interesting.
